android/app/src/main/java/org/prideinlondon/festival/MainApplication.java

```diff
@@ -33,13 +40,13 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
-            BugsnagReactNative.getPackage(),
-            new RNReactNativeHapticFeedbackPackage(),
-            new LottiePackage(),
             new SplashScreenReactPackage(),
             new MapsPackage(),
             new LinearGradientPackage(),
+            new RNReactNativeHapticFeedbackPackage(),
```

Comment:
If there's no significance to this reordering and it was just from runnning RN link then please revert.